core/controllers/contributor_dashboard_admin_test.py

```diff
@@ -845,7 +846,7 @@ def setUp(self) -> None:
             rejected_translations_count=self.REJECTED_TRANSLATIONS_COUNT,
             rejected_translation_word_count=(
                 self.REJECTED_TRANSLATION_WORD_COUNT),
-            first_contribution_date=datetime.datetime.utcnow(),
```

Comment:
Are these changes needed for this PR? Please leave *only changes related to the issue* you are fixing.

Reply:
These are just the enhancements and not the absolute requirements. I real world scenario `first_contribution_date` will always be less than or equal to `last_contribution_date`. For this reason instead of setting `first_contribution_date` as `.utcnow()`, I decided to set it as some date older than `last_contribution_date`.

Reply:
Please remove any unnecessary change from this PR including this. You can add a follow up PR with only these changes and it will be approved smoothly